example-context/example-service/example-event/example-event-listener/pom.xml

```diff
@@ -23,7 +23,11 @@
             <artifactId>javaee-api</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Could we add this  dependency (as it is provided) to the component modules?, so if anybody does want to start using the pool they don't need to add it.

Also presumably this works for Event-Processor, Command Handler MDBs too?